Backends/PostScript/output-destination.lisp

```diff
@@ -0,0 +1,37 @@
+;;; -*- Mode: Lisp; Package: CLIM-POSTSCRIPT -*-
+
+;;;  (c) copyright 1998,1999,2000 by Michael McDonald (mikemac@mikemac.com)
+;;;  (c) copyright 2000 by Robert Strandh (strandh@labri.u-bordeaux.fr)
+;;;  (c) copyright 2002 by Tim Moore (moore@bricoworks.com)
+
+;;; This library is free software; you can redistribute it and/or
+;;; modify it under the terms of the GNU Library General Public
+;;; License as published by the Free Software Foundation; either
+;;; version 2 of the License, or (at your option) any later version.
+;;;
+;;; This library is distributed in the hope that it will be useful,
+;;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;;; Library General Public License for more details.
+;;;
+;;; You should have received a copy of the GNU Library General Public
+;;; License along with this library; if not, write to the
+;;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;;; Boston, MA  02111-1307  USA.
+
+(cl:in-package #:clim-postscript)
+
+(defclass postscript-destination (climi::file-destination)
+  ())
+
+(defmethod climi::invoke-with-standard-output
+    (continuation (destination postscript-destination))
+  (call-next-method (lambda ()
+                      (with-output-to-postscript-stream
+                          (ps-stream *standard-output*)
+                        (let ((*standard-output* ps-stream))
+                          (funcall continuation))))
+                    destination))
+
+(pushnew '("Postscript File" postscript-destination)
+         climi::*output-destination-types* :test #'equal)
```

Comment:
If we move it from core `file-destination` and `invoke-with-standard-output` should be exported from climb package (and at least mentioned as a comment in `developer-manual.texi` as something worth documenting).

Also instead of `pushnew` we should have a function: `register-destination-type` (also in climb package and also at least mentioned in the manual in a comment).

Reply:
I wonder if we should also make available in climb `output-destination` and `stream-destination` classes.

Reply:
Done except for the "If we move it from core …" part which i don't understand.

Reply:
Given we move this snippet from modules in `Core/` directory and we still depend on internal symbols from `CLIMI` package, then these symbols should be promoted to be exported from `CLIMB` package and used that way. We don't want backends to depend on `CLIMI` internal symbols.

Reply:
In the latest version, the symbols are exported from the `climb` package.